nuxeo-java-client/src/main/java/org/nuxeo/client/api/objects/Document.java

```diff
@@ -278,6 +289,9 @@ public String getName() {
     }
 
     public void set(String key, Object value) {
+        if (value instanceof Calendar || value instanceof Date) {
```

Comment:
Why not do `value != null && !(value instanceof String)` instead, as this seems to be the only format supported?